<?xml version="1.0"?>
<launch> 

  <!-- RVIZ -->
  <node name="rviz" pkg="rviz" type="rviz" args="-d $(find jugnoo_gps)/launch/aruco_config.rviz" />

  <!-- ArUco mapping -->
  <node pkg="jugnoo_gps" type="jugnoo_gps" name="jugnoo_gps" output="screen">
    <remap from="/image_raw" to="/usb_cam/image_raw"/>

    <param name="calibration_file" type="string" value="/home/rohan/ost.ini" /> 
    <param name="num_of_markers" type="int" value="10" />
    <param name="marker_size" type="double" value="0.197"/>
    <param name="space_type" type="string" value="plane" />
    <param name="roi_allowed" type="bool" value="false" /> -->
    <param name="roi_x" type="int" value="0" /> -->
    <param name="roi_y" type="int" value="0" /> -->
    <param name="roi_width" type="int" value="640" /> -->
    <param name="roi_height" type="int" value="480" /> --> 

  </node>  
</launch>
